svgr.config.js

```diff
@@ -3,7 +3,10 @@ const fs = require('fs');
 const path = require('path');
 
 const templateCreateStyledIcon = fs.readFileSync(
-  path.join(__dirname, `templates/icon.styles.tsx`),
+  path.join(
+    __dirname,
+    `packages/components/icons/src/templates/icon.styles.tsx`
```

Comment:
Moved this into the `icons` package to be able to export a bunch of stuff needed by the new component.